docs/_scripts/sync-collection.sh

```diff
@@ -0,0 +1,9 @@
+# Sync repository foo using remote cbar
+pulp ansible repository sync --name "foo" --remote "cbar"
+
+# Use the -b option to have the sync task complete in the background
+# e.g. pulp -b ansible repository sync --name "foo" --remote "cbar"
+
+# After the task is complete, it gives us a new repository version
+# Inspecting new repository version
+pulp ansible repository version --repository "foo" show --version 1
```

Comment:
is it possible to display all versions? (if not, maybe we need different repos for testing roles/collections)
since we sync roles first, version 1 is displaying role content on CI

Reply:
Yes, there is a list command that would show all of them. I kept this at version 1 so that users could copy and paste the commands if they are following the collection workflow, but since I didn't create a new repo the CI would be showing the wrong version.  Maybe I should create a PR for the CLI to allow the show command to accept no version and default to showing the latest.

Reply:
+1 to showing the latest command!